.github/workflows/ci.yml

```diff
@@ -161,7 +161,7 @@ jobs:
         pytest pandas/tests/arithmetic/
         pytest pandas/tests/groupby/
         pytest pandas/tests/resample/
-        pytest pandas/tests/reshape/merge
+        pytest pandas/tests/reshape/
```

Comment:
nice!  let's use a glob pattern here and get a bunch of these down to a single line

Reply:
youve got pandas/tests/generic/test_generic.py and pandas/tests/generic/ below

Reply:
This is replaced by `run_tests.sh` now in anther PR, so marking as resolved.